templates/blog/post_detail.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>{{ post.title }}</h1>
+  <p>{{ post.owner.first_name }} {{ post.owner.last_name }} at {{ post.created_time }}</p>
+  <p>{{ post.content }}</p>
+
+  <p>{{ post.commentary.count }} comment{{ post.commentary.count|pluralize }}</p>
+  {% if not post.commentary.exists %}
```

Comment:
The condition `if not post.commentary.exists` should be `if not post.commentary.exists()`. The `exists()` method is a function and needs to be called with parentheses.